ros1_bridge/__init__.py

```diff
@@ -606,7 +606,7 @@ def determine_common_services(
                 ros2_type = str(ros2_fields[direction][i].type)
                 ros1_name = ros1_field[1]
                 ros2_name = ros2_fields[direction][i].name
-                if ros1_type != ros2_type or ros1_name != ros2_name:
+                if ros1_type != ros2_type:
```

Comment:
according to https://github.com/ros2/ros1_bridge/blob/master/doc/index.rst, this is not specification. it should not be checked here.